src/components/Button/Button.module.scss

```diff
@@ -1,6 +1,7 @@
 @import "../../scss/variables";
 .component {
-    a {
+    a,
+    button {
```

Comment:
I feel that giving a class name to the a/button elements could drasitcally simplify this stylesheet.

Also, while I realise that this may not be your intention <rant>anchors and buttons are not the same thing</rant>